### PR TITLE
Installation next steps

### PR DESCRIPTION
**Pull request by aaronmarkham**
## Changes
- Removed reference to the old beta website
- Updated link to the real getting started page for MXNet, not the old one
- Updated link to the real validate MXNet page, not the old one
- Updated link to the real getting started crash course page, not the old one.
- Updated link to the real MXNet API pages, not the old one.

## Additions
- Added relevant next steps. (Currently it is sending you to MXNet's website for more options and API and the crash course.) It is a good user experience to have a Call To Action at the end of the page, and have the CTA be the most relevant thing a new user might do.

### CODE REVIEW

**Comment by zhanghang1989** [comment]
image
Thanks Aaron. 
I don't think the extra links to AutoGluon API doc and Tutorials are necessary, because they are already linked on the right top corner of this page. See the attached figure

**Comment by aaronmarkham** [comment]
> image
> Thanks Aaron.
> I don't think the extra links to AutoGluon API doc and Tutorials are necessary, because they are already linked on the right top corner of this page. See the attached figure

Ya, I know, but I disagree. I updated the notes for the PR for more context.

**Comment by szha** [comment]
Job PR-143-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-143/3/index.html

**Comment by szha** [comment]
Job PR-143-4 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-143/4/index.html

**Comment by jwmueller** [comment]
We agreed to hold off on this for now, since we don't want to overwhelm users with MXNet/Gluon-specific information as this knowledge is not a prerequisite for using autogluon.

**Comment by aaronmarkham** [comment]
Please don't leave beta links on the site.

File: docs/install.rst [hunk 1]
```diff
@@ -12,12 +12,13 @@ Installation
 Next steps
 ~~~~~~~~~~
 
-- Checkout `beta.mxnet.io <http://beta.mxnet.io/install/index.html>`_ for more options such as ARM devices and docker images.
-- `Verify your MXNet installation <https://beta.mxnet.io/install/validate-mxnet.html>`_
+- `AutoGluon tutorials </tutorials/index.html>`_
+- `AutoGluon API </api/index.html>`_
+- Checkout the `MXNet website <https://mxnet.apache.org/get_started>`_ for more installation options such as ARM devices and docker images.
+- `Verify your MXNet installation <https://mxnet.apache.org/get_started/validate_mxnet>`_
```
Comment (aaronmarkham) [inline docs/install.rst]:
```suggestion
```

File: docs/install.rst [hunk 2]
```diff
@@ -12,12 +12,13 @@ Installation
 Next steps
 ~~~~~~~~~~
 
-- Checkout `beta.mxnet.io <http://beta.mxnet.io/install/index.html>`_ for more options such as ARM devices and docker images.
-- `Verify your MXNet installation <https://beta.mxnet.io/install/validate-mxnet.html>`_
+- `AutoGluon tutorials </tutorials/index.html>`_
+- `AutoGluon API </api/index.html>`_
+- Checkout the `MXNet website <https://mxnet.apache.org/get_started>`_ for more installation options such as ARM devices and docker images.
```
Comment (aaronmarkham) [inline docs/install.rst]:
```suggestion
```

File: docs/install.rst [hunk 3]
```diff
@@ -12,12 +12,13 @@ Installation
...
+- `Verify your MXNet installation <https://mxnet.apache.org/get_started/validate_mxnet>`_
 - `Configure MXNet environment variables <https://mxnet.incubator.apache.org/versions/master/faq/env_var.html>`_
-- For new users: `60-minute Gluon crash course <https://beta.mxnet.io/guide/getting-started/crash-course/index.html>`_
-- For experienced users: `MXNet Guides. <https://beta.mxnet.io/guide/getting-started/crash-course/index.html>`_
-- For advanced users: `MXNet API <https://beta.mxnet.io/api/index.html>`_ and `GluonCV API <../api/index.html>`_.
+- For new Gluon users: `60-minute Gluon crash course <https://mxnet.apache.org/api/python/docs/tutorials/getting-started/crash-course/index.html>`_
```
Comment (aaronmarkham) [inline docs/install.rst]:
```suggestion
```

File: docs/install.rst [hunk 4]
```diff
@@ -12,12 +12,13 @@ Installation
...
+- For advanced users: `MXNet API <https://mxnet.apache.org/api/python/docs/api/index.html>`_ and the `MXNet ecosystem <https://mxnet.apache.org/ecosystem>`_.
```
Comment (aaronmarkham) [inline docs/install.rst]:
```suggestion
```

File: docs/install.rst [hunk 5]
```diff
@@ -12,12 +12,9 @@ Installation
 Next steps
 ~~~~~~~~~~
 
-- Checkout `beta.mxnet.io <http://beta.mxnet.io/install/index.html>`_ for more options such as ARM devices and docker images.
```
Comment (zhanghang1989) [inline docs/install.rst]: Could you update the links instead of deleting them?

**Comment by szha** [comment]
Job PR-143-5 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-143/5/index.html

**Comment by szha** [comment]
Job PR-143-6 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-143/6/index.html

File: docs/install.rst (hunk 5 above)
Comment (aaronmarkham) [reply docs/install.rst]: Uh... that's what the original PR did but y'all said there was too much MXNet.

Comment (zhanghang1989) [reply docs/install.rst]: I think we can probably keep crush course.

**Comment by szha** [comment]
Job PR-143-7 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-143/7/index.html

**Comment by szha** [comment]
Job PR-143-8 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-143/8/index.html

**Pull request closed without merging**